examples/textures/01-logo-raylib.lisp

```diff
@@ -0,0 +1,28 @@
+(in-package #:claylib/examples)
+
+(defun example-textures-01 ()
+  (with-window (:title "raylib [textures] example - texture loading and drawing")
+    (let* ((image-size 256)
+           (source-rec (make-rectangle 0 0 image-size image-size +white+))
+           (dest-rec (make-rectangle (truncate (- (get-screen-width) image-size) 2)
+                                     (truncate (- (get-screen-height) image-size) 2)
+                                     image-size
+                                     image-size +white+))
+           (scene (make-scene `((texture ,(make-instance
+                                           'texture
+                                           :path "resources/raylib_logo.png"
+                                           :origin (make-vector2 0 0)
+                                           :source source-rec
+                                           :dest dest-rec
+                                           :tint +white+
+                                           :rot 0.0)))
+                              `((text ,(make-text "this IS a texture!"
+                                                  360 370
+                                                  :size 10
+                                                  :color +gray+))))))
+      (with-scene scene ()
+        (with-scene-assets (texture) scene
+          (do-game-loop ()
+            (with-drawing
+              (draw-scene-all scene)
+              (draw-object texture))))))))
```

Comment:
Should `draw-scene-all` also render assets?

Reply:
Also it seems to me that one might want `draw-object` for textures to call upon different raylib functions like `DrawTextureTiled`. Should we support that with a keyword? It makes the texture class a bit clunkier if we're going to add slots for all the data needed for the different ways to draw.

Reply:
I'll look into this and get back to you.